frontend/src/app/modules/login/containers/login-screen/login-screen.component.ts

```diff
@@ -29,6 +32,14 @@ export class LoginScreenComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    this.subscription = this.count$.subscribe();
+    console.log(this.subscription);
```

Comment:
Do we need to track something here?

Reply:
removed. Please fix with new structure existed redux scenario (it's good case to start working with ngrx)

Reply:
It is not working right now with the new structure, rigth?